modules/index-service/src/test/java/org/opencastproject/index/service/impl/IndexServiceImplTest.java

```diff
@@ -124,14 +128,14 @@
 public class IndexServiceImplTest {
 
   private static final JSONParser parser = new JSONParser();
+  private static final Logger logger = LoggerFactory.getLogger(IndexServiceImplTest.class);
 
   /**
    * Tests for the method calculatePeriods
    */
-  private final TimeZone utc = TimeZone.getTimeZone("UTC");
-  private final TimeZone jst = TimeZone.getTimeZone("JST"); // Japan Standard Time (UTC +9)
-  private final TimeZone pst = TimeZone.getTimeZone("PST"); // Alaska Standard Time (UTC -8)
-  private final TimeZone cet = TimeZone.getTimeZone("CET"); // Alaska Standard Time (UTC +2)
+  private final TimeZone jst = TimeZone.getTimeZone("Asia/Tokyo"); // Japan Standard Time (UTC +9)
+  private final TimeZone pst = TimeZone.getTimeZone("America/Anchorage"); // Alaska Standard Time (UTC -8)
+  private final TimeZone cet = TimeZone.getTimeZone("Europe/Zurich"); // European time (UTC +2)
```

Comment:
@gregorydlogan @KatrinIhler the unit tests in this pull work when I change my system to be in a non-DST observing TimeZone like "America/Phoenix". But, I'll also add that non-DST boundry timeZone to these tests.

Reply:
FYI - I tested this pull from Phoenix time zone (non Daylight Savings TZ) and added junit year+ bulk scheduling to the Phoenix time zone but could not reproduce the issue @gregorydlogan logged at https://github.com/ical4j/ical4j/issues/232 using this pull.